docs/migration/v8-to-v9.md

```diff
@@ -70,6 +70,10 @@ Sentry.init({
 
 - When `skipOpenTelemetrySetup: true` is configured, `httpIntegration({ spans: false })` will be configured by default. This means that you no longer have to specify this yourself in this scenario. With this change, no spans are emitted once `skipOpenTelemetrySetup: true` is configured, without any further configuration being needed.
 
+### `@sentry/react`
+
+The `componentStack` field in the `ErrorBoundary` component is now typed as `string | undefined` instead of `string | null | undefined`. This more closely matches the actual behavior of React, which always returns a `string` whenever a component stack is available. `undefined` is only returned if no error has been caught by the error boundary.
```

Comment:
m: Looking at the code, I don't see `componentStack` being typed as `string | undefined` but rather `string | null`. Should we update the migration note accordingly?

```suggestion
The `componentStack` field in the `ErrorBoundary` component is now typed as `string | null` instead of `string | null | undefined`. This more closely matches the actual behavior of React, which always returns a `string` whenever a component stack is available. `undefined` is only returned if no error has been caught by the error boundary.
```

i might just also be missing something 😅 - in that case, feel free to disregard!